pkg/nsx/client.go

```diff
@@ -18,13 +20,12 @@ import (
 	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra/domains"
 	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra/domains/security_policies"
 	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra/sites/enforcement_points"
+	vpcs "github.com/vmware/vsphere-automation-sdk-go/services/nsxt/orgs/projects"
```

Comment:
It will be confused when import this package as vpcs, because there is a 'real' vpcs package in it.
""github.com/vmware/vsphere-automation-sdk-go/services/nsxt/orgs/projects/vpcs""